pkg/prometheus/promcfg.go

```diff
@@ -1611,16 +1611,17 @@ func generateRunningFilter() yaml.MapSlice {
 	}
 }
 
-func getLimit(user *uint64, enforced *uint64) *uint64 {
+func (cg *ConfigGenerator) getLimit(user *uint64, enforced *uint64) *uint64 {
 	if enforced == nil {
```

Comment:
(suggestion) easier to use ptr.Deref() from the start?
```suggestion
	if ptr.Deref(enforced, 0) == 0 {
```